core/src/main/scala/kafka/admin/TopicCommand.scala

```diff
@@ -22,7 +22,7 @@ import java.util.{Collections, Properties}
 import joptsimple._
 import kafka.common.AdminCommandFailedException
 import kafka.log.LogConfig
-import kafka.utils._
+import kafka.utils.{immutable => _, _}
```

Comment:
Changes like this are due to shadowing between `kafka.utils.immutable` and the `immutable` package in `scala.collections`.